[FEATURE] DOCKERIZE & FIX GAMETIME

cjuroz opened the pull request:
- Dockerize l1j-en to be executed as docker compose service (db, server, phpmyadmin, nginx).
- Fix table names from m10 script.
- Add GameTimeShift in server.properties to trick gametime back to 2013.
- Set slf4j-jul binding and log.properties.
- Fix broken newbie quests (Teo/Ruba).
- Kick players from newbie areas conistently with db (at level 13).
- Enable Arena guards in singing island.

bm3719 commented:
Would be happy to merge this with one possible caveat: The changes to config/server.properties in 04b03c2614ed42beb8401047bb1a56921e9a6ee4 would default everyone to use Docker.

It's nice to know it runs fine in Docker, but pretty sure most people wouldn't want to do that.  Maybe there could be a docker-specific version of this that gets COPY-ed over to the image version?

cjuroz commented:
You are right! Reverted server.properties back to original so people can keep running **l1j** as a `normal` application and updated docker-compose.yaml to use an environment variable in case people prefer to run everything as a `docker` application. Changes are transparent to both type of users.

bm3719 commented:
Awesome, thanks.  I'll send you an invite in case you wanna do any further contributions to the project.  Feel free to accept or decline in accordance with your interest level.  At the moment, none of the other members are actively working on this, including myself.
